src/NuGet.Core/NuGet.Packaging/Signing/Utility/SignatureUtility.cs

```diff
@@ -656,6 +652,26 @@ private static DerSequenceReader CreateDerSequenceReader(CryptographicAttributeO
             return new DerSequenceReader(attribute.Values[0].RawData);
         }
 
+
+        private static byte[] FromHexStringToByteArray(string thumbprint)
+        {
+            byte[] hash = new byte[thumbprint.Length / 2];
+            for (int i = 0; i < hash.Length; i++)
+            {
+                string byteString = thumbprint.Substring(i * 2, 2);
```

Comment:
Use named parameters `startIndex` and `length`.

Reply:
Thanks. Fixed.